autoload/scratch.vim

```diff
@@ -50,6 +59,12 @@ function! s:open_window(position)
 endfunction
 
 function! s:close_window(force)
+
```

Comment:
Could you move the comment back to the top of the function definition and remove the empty lines for consistency (same for the other diff above)?